pkg/catalog/grpc_translators.go

```diff
@@ -0,0 +1,11 @@
+package catalog
+
+import profilesv1 "github.com/weaveworks/profiles/api/v1alpha1"
+
+type grpcProfileCatalogEntry struct {
+	Item profilesv1.ProfileCatalogEntry
+}
+
+type grpcProfileCatalogEntryList struct {
+	Items []profilesv1.ProfileCatalogEntry
+}
```

Comment:
does it make sense for this to live in PCTL or profiles?

Reply:
saw this defined in some acceptance tests in profiles too

Reply:
Yes, because both do json unmarshalling. I'm not sure profile should care how users of profile marshal the returned data 🤔

Reply:
Well profiles is dictating what the format is right?

Reply:
Hm. Ah, what the hell. I can add it there, sure. :D

Reply:
Extracted https://github.com/weaveworks/profiles/pull/184/commits/7e36e218f9444696f6836c046cd5f091ed578274 to use here as well.